api/authentication/selectors.py

```diff
@@ -0,0 +1,19 @@
+from typing import Union
+
+from rest_framework.exceptions import ValidationError
+
+from api.users.models import User
+
+
+def validate_phone(*, phone: str) -> Union[None, ValidationError]:
```

Comment:
I am not sure that we include exceptions in the function type hint

Reply:
No problem, other than that all good? In terms of following the style guide